webpack.config.js

```diff
@@ -0,0 +1,31 @@
+// const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  entry: './src/index.js',
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
+    }),
+  ],
+  // output: {
+  //   filename: '[main].bundle.js',
+  //   path: path.resolve(__dirname, 'dist'),
+  //   clean: true,
+  // },
```

Comment:
It's part of the best practices to not keep commented code blocks in your codebase. Kindly make sure to remove it. It'll help you to have a cleaner repository.